test/simplify_reshapes_test.cpp

```diff
@@ -337,6 +337,115 @@ TEST_CASE(nop_convert)
     EXPECT(std::distance(m.begin(), m.end()) == n - 1);
 }
 
+TEST_CASE(concat_multibroadcasts1)
+{
+    // Broadcasted batch dim, new axis < old axis
+    migraphx::module m;
+
+    auto s  = migraphx::shape{migraphx::shape::float_type, {3, 4}};
```

Comment:
All of these tests have the same program pattern. I suggest a function like this:

```
migraphx::module make_concat_multibroadcast(std::vector<size_t> in_shape, std::vector size_t mbcast_shape, int axis)
{
    migraphx::module m;
    auto s  = migraphx::shape{migraphx::shape::float_type, in_shape};
    auto x  = m.add_parameter("x", s);
    auto y  = m.add_parameter("y", s);
    auto z  = m.add_parameter("z", s);
    auto xm = m.add_instruction(migraphx::make_op("multibroadcast", {{"out_lens", mbcast_shape}}), x);
    auto ym = m.add_instruction(migraphx::make_op("multibroadcast", {{"out_lens", mbcast_shape}}), y);
    auto zm = m.add_instruction(migraphx::make_op("multibroadcast", {{"out_lens", mbcast_shape}}), z);
    auto concat = m.add_instruction(migraphx::make_op("concat", {{"axis", axis}}), xm, ym, zm);
    m.add_return({concat});
    return m;
}